src/content/App/App.tsx

```diff
@@ -71,6 +71,12 @@ function App() {
           //Don't set the term to WF, just keep the term to what is selected
           setCurrentTerm(newData.terms.values().next().value)
         }
+      } else if (changes.sections) {
```

Comment:
Makes sense, I am going to leave it out of this PR for the moment since it may affect potential state interactions. I will look into it further once this PR merges